subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c

```diff
@@ -190,7 +190,7 @@ static void pu_reset_timing_restrict(struct ll_conn *conn)
 }
 
 #if defined(CONFIG_BT_PERIPHERAL)
-static inline bool phy_valid(uint8_t phy)
+static inline bool phy_validation(uint8_t phy)
```

Comment:
It is a different PHY validation, see LL_PHY_CHECK_IND: 

_"If the Peripheral receives an LL_PHY_UPDATE_IND where either PHY field specifies a PHY that the Peripheral does not support, has a bit set that is reserved for future use, or has more than one bit set, the Peripheral shall not change the PHY in that direction."_

For LL_PERIODIC_SYNC_IND is: 
_"However, if the PHY field of the LL_PERIODIC_SYNC_IND PDU or the LL_PERIODIC_SYNC_WR_IND PDU has no bits or more than one bit set, or the bit set corresponds to a PHY that the recipient does not support or is reserved for future use, the recipient shall ignore the PDU"_

Reply:
Aha. 

We should not have a function called `phy_valid` that works on one set of PHY values, and a function `phy_validation` that works on another set of PHY values. 

The function names should indicate the behavior of them, and since they both have the same name and different behavior, that's just a bug waiting to happen

Reply:
The `phy_validation` is static and can only be used in ull_llcp_phy.c 

Where `phy_valid` is shared for several procedures as it is a more general check. However, I get that the name can give problmes. I can try to get up with a better name for the ` ull_llcp_phy.c ` specific version.

Reply:
Thanks :)